distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -50,6 +50,7 @@
 import ddf.common.test.callables.GetConfigurationProperties;
 import ddf.common.test.matchers.ConfigurationPropertiesEqualTo;
 import ddf.test.itests.AbstractIntegrationTest;
+import jdk.nashorn.internal.ir.annotations.Ignore;
```

Comment:
Whoops. The next commit won't have anyway Ignores so I'll make sure to remove this.